src/charactersheet/viewmodels/dm/encounter_sections/npc_section/index.html

```diff
@@ -111,7 +112,7 @@
                         hasFocus: firstElementInModalHasFocus">
             </div>
             <div class="col-sm-1 control-label">
-              <span class="fa fa-paper-plane-o" style="cursor:pointer;"
+              <span class="fa fa fa-desktop" style="cursor:pointer;"
                 title="This field will be sent to the players via the chat.">
```

Comment:
since we're not having chat now, will this field be shown in the exhibit tab now?

if so, should probably update the text to reflect that. Same for other sections in this file.

Reply:
I can update these, but I've already rewritten the entire UI for the DMs in my other PR, so I'd prefer not to do it twice. Thoughts?

Reply:
sounds good